test/common.js

```diff
@@ -1,9 +1,28 @@
 'use strict';
+
+process.env.NODE_ENV = 'test';
+
 require('dotenv').config();
 const _ = require('lodash');
 const nock = require('nock');
 const apiServer = require('./fixtures/api');
 const fixtures = require('./fixtures');
+const config = require('../config');
+
+function clearDB() {
+  const tables = [
+    'oauth_credentials',
+    'users',
+  ];
+  let deferred = config.bookshelf.knex.migrate.latest();
+
+  for (const tableName of tables) {
+    // eslint-disable-next-line no-loop-func
+    deferred = deferred.then(() => config.bookshelf.knex(tableName).select().del());
```

Comment:
Migration rollbacks are a bit tricky. It's not uncommon for migrations to be destructive ([example in OpenTrials API](https://github.com/opentrials/api/blob/master/migrations/20160415172210_remove_trials_trialrecords_table.js)), and so rolling back wouldn't work. We could drop the entire database and recreate it here, but I haven't found an easy way to do that with KnexJS (and it requires the user to be able to create databases).